tools/test/topology/test-playback.m4

```diff
@@ -32,17 +35,50 @@ DEBUG_START
 # TEST_SSP_PHY_BITS - SSP physical slot size
 # TEST_SSP_DATA_BITS - SSP data slot size
 # TEST_SSP_MODE - SSP mode e.g. I2S, LEFT_J, DSP_A and DSP_B
+# TEST_PIPE_AMOUNT - Total amount of pipelines e.g. 1, 2, 3, 4
 #
 
 # Apply a non-trivial filter blob IIR and FIR tests. TODO: Note that the
 # PIPELINE_FILTERx notation will be updated in future for better flexibility.
 define(PIPELINE_FILTER1, ifelse(TEST_PIPE_NAME, `eq-iir', `eq_iir_coef_loudness.m4'))
 define(PIPELINE_FILTER2, ifelse(TEST_PIPE_NAME, `eq-fir', `eq_fir_coef_loudness.m4'))
 
+# Define HAS_PIPEn flags according to TEST_PIPE_AMOUNT. Those flags will be used to
+# determine whether PIPELINE_n should be added.
+ifelse(TEST_PIPE_AMOUNT, `2',
+`
+define(HAS_PIPE2)
```

Comment:
We probably want to prefix this so that it wont collide with other generic terms.

Reply:
Renamed HAS_PIPEx to TEST_HAS_PIPEx.